client/my-sites/pages/page/index.js

```diff
@@ -73,6 +73,8 @@ class Page extends Component {
 		recordMoreOptions: PropTypes.func.isRequired,
 		recordPageTitle: PropTypes.func.isRequired,
 		recordEditPage: PropTypes.func.isRequired,
+		// Adding new make homepage item
+		recordMakeHomepage: PropTypes.func.isRequired,
```

Comment:
Here and everywhere else, this should change to "Set as front page".  The verb "Make" means too many different things, and as previously noted, the standard WordPress terminology for this setting is "front page".

Reply:
and likewise, I'm happy to go into my branch and make this change as well.